src/app/doc/parts/start/start.component.ts

```diff
@@ -65,8 +66,12 @@ import { NgxSmartModalService } from 'ngx-smart-modal';
   // ...
 })
 export class AppComponent {
+  // If Angular < 8
   @ViewChild(TemplateRef) tpl: TemplateRef<any>;
 
+  // If Angular >= 8
+  //@ViewChild(TemplateRef, { static: false }) tpl: TemplateRef<any>;
+
```

Comment:
It's still correct on all Angular version, this parameter is just optional before v8

Reply:
nope:
![image](https://user-images.githubusercontent.com/5319267/65252290-fc3b4c80-daf8-11e9-85f8-dedabbb26ce4.png)